src/xcode/ENA/ENA/Source/Scenes/CreateAntigenTestProfile/CreateAntigenTestProfileViewModel.swift

```diff
@@ -15,7 +15,14 @@ final class CreateAntigenTestProfileViewModel {
 		self.antigenTestProfile = AntigenTestProfile()
 
 		// this is only for coordinator testing, remove later
```

Comment:
should that be removed or not?

Reply:
At the moment it is required, but should be removed within the create profile branch